### ISSUE TITLE
How to augment image and mask at the same time

### ISSUE THREAD

**Issue by beichuanzhaolun** [MIC-DKFZ/batchgenerators#25]
I was trying to train a 3d Segmentation network

my dataloader yields:
data: torch.Size([4, 1, 128, 128, 128])
label: torch.Size([4, 1, 128, 128, 128])

Then I used crop augmentation, then I get:
data: torch.Size([4, 1, 48, 48, 48])
label: torch.Size([4, 1, 128, 128, 128])

'data' has been croped, but 'label' was not.

How can I crop them samely?

Thanks!
